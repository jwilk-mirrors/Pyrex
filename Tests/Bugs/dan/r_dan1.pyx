import pkg.module as m
