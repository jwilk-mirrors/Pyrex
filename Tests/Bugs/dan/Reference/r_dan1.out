Pyrex: pkg.module
Python: pkg.module
